mdn/templates/mdn/feature_page_list.jinja2

```diff
@@ -98,10 +143,10 @@
 
 {% endblock content %}
 
-{% block block_js_extra %}
+{% block body_js_extra %}
 <script>
 $(function () {
-  $('#issue-header').tooltip()
+  $('[data-toggle="tooltip"]').tooltip()
```

Comment:
I tried to be clever, and it didn't quite work, so I'm back to doing what the [bootstrap docs](http://getbootstrap.com/javascript/#callout-tooltip-opt-in) suggests.